Assignment3/assignment3.txt

```diff
@@ -0,0 +1,12 @@
+Aldin Aksay
+
+1. Determine the complexity
+	a: 24 bytes, O(n)
+	b: 32 bytes, O(n^2 + n)
```

Comment:
Close! since it's big O notation, you don't have to care about the lower terms, so this just becomes O(n^2).